build.gradle

```diff
@@ -275,6 +276,18 @@ subprojects {
   }
   test.dependsOn('checkstyleMain', 'checkstyleTest')
 
+  findbugs {
```

Comment:
So, it seems like Gradle sometimes uses older versions of the plugins (https://github.com/gradle/gradle/issues/1067). Maybe we should define the version ourselves? That way, we can upgrade the version when it suits us instead of having silent tool upgrades when we upgrade the Gradle version.